test/src/receiver/main.cpp

```diff
@@ -28,102 +28,148 @@ using TIME = NDTime;
 
 
 /***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct inp : public cadmium::in_port<Message_t>{};
+struct inp : public cadmium::in_port<Message_t>{
+
+};
 
 /***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct outp : public cadmium::out_port<Message_t>{};
+struct outp : public cadmium::out_port<Message_t>{
+
+};
 
 
 /********************************************/
 /****** APPLICATION GENERATOR *******************/
 /********************************************/
 template<typename T>
 class ApplicationGen : public iestream_input<Message_t,T> {
-public:
-  ApplicationGen() = default;
-  ApplicationGen(const char* file_path) : iestream_input<Message_t,T>(file_path) {}
+    public:
+    ApplicationGen() = default;
```

Comment:
Indentation issue: There should be a tab for all the statements following the public keyword declaration. 
i.e,
public:
      ApplicationGen() = default;
      ........
      ........

Reply:
done